examples/data_types_and_io/data_types_and_io/attribute_access.py

```diff
@@ -0,0 +1,158 @@
+# %% [markdown]
+# (attribute_access)=
+#
+# # Attribute Access
+#
+# ```{eval-rst}
+# .. tags:: Basic
+# ```
+#
+# Flyte allows users to access attributes directly on output promises for List, Dict, Dataclass, and combinations of them. This allows users to pass attributes of the output directly in workflows, making it more convenient to work with complex data structures.
+#
+# To begin, import the necessary dependencies and define a common task for later use.
+# %%
+from dataclasses import dataclass
+from typing import Dict, List
+
+from dataclasses_json import dataclass_json
+from flytekit import WorkflowFailurePolicy, task, workflow
+
+
+@task
+def print_str(a: str):
+    print(a)
+    return
+
+
+# %% [markdown]
+# ## List
+# We can access the output list by index.
```

Comment:
I recommend "You can access" instead of "We can access" for consistency with "You can run the workflow locally"